spacy/cli/debug_data.py

```diff
@@ -193,6 +194,66 @@ def debug_data(
     else:
         msg.info("No word vectors present in the package")
 
+    if "spancat" in factory_names:
+        model_labels = _get_labels_from_model(nlp, "spancat")
+        labels = [list(l.keys()) for l in gold_train_data["spancat"].values()]
+        has_low_data_warning = False
+        has_no_neg_warning = False
+        has_ws_ents_error = False
+
+        msg.divider("Span Categorization")
+        msg.info(f"{len(model_labels)} label(s)")
+
+        # Check if some model labels are not present in actual dataset
+        msg.text("Labels in train data: ")
+        for span_key, data_labels in gold_train_data["spancat"].items():
```

Comment:
Can you use `spans_key` instead of `span_key` to be consistent with `spancat`?

Reply:
Updated in https://github.com/explosion/spaCy/pull/10070/commits/8e2d2e209f8e1619e1fbdcfed6f200ef919fdb1f